source/simulator/parameters.cc

```diff
@@ -1139,63 +1139,96 @@ namespace aspect
                            boost::lexical_cast<std::string>(std::numeric_limits<double>::lowest()),
                            Patterns::List(Patterns::Double ()),
                            "The minimum global composition value that will be used in the bound preserving "
                            "limiter for the discontinuous solutions from composition advection fields. "
                            "The number of the input 'Global composition minimum' values separated by ',' has to be "
                            "one or the same as the number of the compositional fields. When only one value "
-                           "is supplied, this same value is assumed for all compositional fields.");
+                           "is supplied, this same value is assumed for all compositional fields.\n"
+                           "The input value is active only when `Limiter for discontinuous composition "
+                           "solution' is set to `bound preserving'.");
+        prm.declare_entry ("Temperature KXRCF indicator threshold", "1.0",
+                           Patterns::Double(0.0),
+                           "The threshold of KXRCF indicator for the temperature field, as described in "
+                           "\\cite{Krivodonova:etal:2004}. If the KXRCF indicator of a cell is greater than "
+                           "the threshold, then the cell is marked as 'troubled' and will be smoothed by "
+                           "the WENO limiter.\n"
+                           "The input value is active only when `Limiter for discontinuous temperature "
+                           "solution' is set to `WENO'.");
+        prm.declare_entry ("Composition KXRCF indicator threshold", "1.0",
+                           Patterns::Double(0.0),
+                           "The threshold of KXRCF indicator for the temperature field, as described in "
+                           "\\cite{Krivodonova:etal:2004}. If the KXRCF indicator of a cell is greater than "
+                           "the threshold, then the cell is marked as 'troubled' and will be smoothed by "
+                           "the WENO limiter. The number of the input values separated by ',' has to be one "
+                           "or the same as the number of the compositional fields. When only one value is "
+                           "supplied, this same value is assumed for all compositional fields.\n"
+                           "The input value is active only when `Limiter for discontinuous composition "
+                           "solution' is set to `WENO'.");
+        prm.declare_entry ("WENO linear weight of neighbor cells", "0.001",
+                           Patterns::Double(0.0, 1.0),
+                           "The linear weight of neighbor cells in WENO scheme. The larger this value is, "
+                           "the more information of neighbor cells will be involved in the polynomial "
```

Comment:
I havent read the paper, but I think this wording is a bit misleading. You likely mean:
```suggestion
                           "the more weight is given to information from neighbor cells when reconstructing the polynomial "
```
If you say `more information` that implies that you include additional sources of information (e.g. more degrees of freedom or additional fields), but that is likely not what you mean.